src/test/java/Homework17.java

```diff
@@ -0,0 +1,32 @@
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+public class Homework17 extends BaseTest{
+
+    @Test
+    public void addSongToPlaylist() throws InterruptedException {
+
+        String expectedSongAddedMessage = "Added 1 song into \"Test Pro Playlist.\"";
+
+        navigateToPage();
+        provideEmail("demo@class.com");
+       providePassword("te$t$tudent");
+        clickSubmit();
+        Thread.sleep(2000);
+        searchSong("Pluto");
+        Thread.sleep(2000);
+        clickViewAllBtn();
+       selectFirstSongResult();
+       clickAddToBtn();
+        choosePlaylist();
+        Assert.assertEquals(getAddToPlaylistSuccessMessage(), expectedSongAddedMessage);
+    }
+
+
+    }
+
+
+
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._